packages/core/src/list-transactions.ts

```diff
@@ -95,8 +95,10 @@ export async function listTransactions(
                 ? exState.result.address
                 : undefined
               : undefined,
-          params: exState.constructorArgs,
-          value: networkInteraction.value,
+          params: exState.constructorArgs.map((arg) =>
```

Comment:
The type of `params` is `SolidityParameterType[]`, which supports `bigint`.

I think we should leave the mapping here as is and deal with the issue of bigin serialization in `JSON.stringify` in:

https://github.com/NomicFoundation/hardhat-ignition/blob/44ebf75ad3d0ef47f6acc7022ceac03f0e8454c0/packages/hardhat-plugin/src/ui/helpers/calculate-list-transactions-display.ts#L30